### PR TITLE
fix(search): ignore header and footer in search index, fixes #265

### LINKED ISSUE #265: Global header is indexed for documentation pages
When I search the Anbox Cloud docs for "security", I get (after some expected and useful results) every other page with the snippet "The new standard secure enterprise Linux for servers, desktops, clouds, ... Extended Security Maintenance, Kernel Livepatch, FIPS, enterprise support and ..." - which apparently comes from the global page header.

That header should not be indexed for documentation pages (and probably not for other pages either), because it pollutes the search results.

### PR DESCRIPTION
**Pull request by edlerd**
## Done
Marked header and footer with `data-nosnippet` attribute to ignore them in Google search index. Should fix #265

### CODE REVIEW

**Comment by edlerd** [comment]
Draft because waiting for merge in global-nav project, which we depend on: https://github.com/canonical/global-nav/pull/231
Need to update dependency once it lands

**Review by yurii-vasyliev — APPROVED** [review]

**Pull request merged**
